rust_src/src/lisp.rs

```diff
@@ -699,6 +711,12 @@ impl TailsIter {
         }
     }
 
+    pub fn current(&self) -> LispObject {
```

Comment:
Hmm, `current` is not an intuitive name for this. Mabye `rest`?

Reply:
Hmmm, maybe.